wp-modules/editor/js/src/components/SidebarPanels/CategoriesPanel.tsx

```diff
@@ -20,6 +25,9 @@ export default function CategoriesPanel( {
 	handleChange,
 }: BaseSidebarProps< 'categories' | 'customCategories' > &
 	AdditionalSidebarProps< 'categoryOptions' > ) {
+	const [ categoryTitleIsInvalid, setCategoryTitleIsInvalid ] =
+		useState( false );
```

Comment:
I couldn't think of a simple way to show the modified style (the `styles` prop in `Creatable` below) without using state.